features/tag_filters.feature

```diff
@@ -12,19 +12,13 @@ Feature: Tags
 
       class FeatureContext implements Context
       {
-          /**
-           * @Given /^Some slow step N(\d+)$/
-           */
+          #[Given('/^Some slow step N(\d+)$/')]
```

Comment:
you forgot adding the use statement for the attributes in those files, which breaks things